neqo-transport/tests/server.rs

```diff
@@ -1109,8 +1108,8 @@ fn max_streams_after_0rtt_rejection() {
         Box::new(RejectZeroRtt {}),
         Rc::new(RefCell::new(CountingConnectionIdGenerator::default())),
         ConnectionParameters::default()
-            .max_streams(StreamType::BiDi, StreamIndex::new(MAX_STREAMS))
-            .max_streams(StreamType::UniDi, StreamIndex::new(MAX_STREAMS)),
+            .max_streams(StreamType::BiDi, MAX_STREAMS)
+            .max_streams(StreamType::UniDi, MAX_STREAMS),
```

Comment:
One thing to think about with these is the possibility that values can get switched.  It might be worth using different values for unidirectional and bidirectional stream limits in these tests.